templates/workspaces/airlock-import-review/porter.yaml

```diff
@@ -159,10 +157,6 @@ install:
         enable_local_debugging: ${ bundle.parameters.enable_local_debugging }
         register_aad_application: ${ bundle.parameters.register_aad_application }
         create_aad_groups: ${ bundle.parameters.create_aad_groups }
-        arm_tenant_id: ${ bundle.credentials.azure_tenant_id }
```

Comment:
Why are these no longer needed?

Reply:
We used to use the az cli within the base workspace bundle. This is no longer there hence these variables are not required.